design-reference/architecture/VersionAndGo/readme.md

```diff
@@ -0,0 +1,113 @@
+# Version and Go Technical Design Purpose
+
+This design addresses Cobalt's workflow to deploy individual application infrastructure using a Cobalt template.
+
+## Definitions
+ - `Engineering Team (Eng Team)` - DevOps or Operations team consisting of Infrastructure Administrators or similar roles.
+ - `Application Team (App Team)` - Software Developers with an application to deploy to Azure.
+ - `Development Repo (Dev Repo)` - Cobalt's repo hosted on Github at [https://github.com/microsoft/cobal](https://github.com/microsoft/cobal)
+ - `Engineering Repo (Eng Repo)` - Individual template repository managed by the `Engineering Team`
+ - `Application Repo (App Repo)` - Repository that contains the application code to be deployed and managed by the `Application Team`
+ - `Version` - all versioning schemes should use [SemVer](https://semver.org/) or variant. 
+
+## Version and Go Setup Description
+Our design starts after the initial Cobalt setup, where an individual template is cloned into an organizations source control (ref: [Setup Doc](https://github.com/microsoft/cobalt/blob/master/docs/GETTING_STARTED_APP_DEV.md)). An application development team is ready to start deploying their code to infrastructure built in Azure. Typically they would make a request to the engineering team. The Eng team would provide the App team with a link to the latest infra configuration files for a specific template. The App team downloads this tar/zip file and extracts the folder/files.
```

Comment:
> The Eng team would provide the App team with a link to the latest infra configuration files for a specific template. The App team downloads this tar/zip file and extracts the folder/files.

See below for ideas about how to avoid this step, which would lower the burden & manual effort for the engineering teams